doc/source/train/user_guide.rst

```diff
@@ -328,168 +328,184 @@ perform hyperparameter tuning with Ray Train, please refer to the
 
 .. _train-result-object:
 
-The Result Object
------------------
+Accessing Training Results
+--------------------------
 
 .. TODO(ml-team) Flesh this section out.
 
 The return of a ``Trainer.fit`` is a :class:`Result` object, containing
 information about the training run. You can access it to obtain saved checkpoints,
 metrics and other relevant data.
 
-.. _train-log-dir:
+.. automodule:: ray.air.result
+    :members:
+    :noindex:
 
-Log Directory Structure
-~~~~~~~~~~~~~~~~~~~~~~~
+For example, you can:
 
-Each ``Trainer`` will have a local directory created for logs and checkpoints.
+* print the metrics for the last iteration:
 
-You can obtain the path to the directory by accessing the ``log_dir`` attribute
-of the :class:`Result` object returned by ``Trainer.fit``.
-
-.. TODO link to Training Run Iterator API as a 3rd option for logging.
-
-.. _train-monitoring:
+.. code-block:: python
 
-Logging, Monitoring, and Callbacks
-----------------------------------
+    from pprint import pprint
+
+    pprint(results.metrics)
+    # {'_time_this_iter_s': 0.001016855239868164,
+    #  '_timestamp': 1657829125,
+    #  '_training_iteration': 2,
+    #  'config': {},
+    #  'date': '2022-07-14_20-05-25',
+    #  'done': True,
+    #  'episodes_total': None,
+    #  'epoch': 1,
+    #  'experiment_id': '5a3f8b9bf875437881a8ddc7e4dd3340',
+    #  'experiment_tag': '0',
+    #  'hostname': 'ip-172-31-43-110',
+    #  'iterations_since_restore': 2,
+    #  'node_ip': '172.31.43.110',
+    #  'pid': 654068,
+    #  'time_since_restore': 3.4353830814361572,
+    #  'time_this_iter_s': 0.00809168815612793,
+    #  'time_total_s': 3.4353830814361572,
+    #  'timestamp': 1657829125,
+    #  'timesteps_since_restore': 0,
+    #  'timesteps_total': None,
+    #  'training_iteration': 2,
+    #  'trial_id': '4913f_00000',
+    #  'warmup_time': 0.003167867660522461}
+
+* View the dataframe containing the metrics from all iterations:
 
-Ray Train has mechanisms to easily collect intermediate results from the training workers during the training run
-and also has a :ref:`Callback interface <train-callbacks>` to perform actions on these intermediate results (such as logging, aggregations, etc.).
-You can use either the :ref:`built-in callbacks <air-builtin-callbacks>` that Ray AIR provides,
-or implement a :ref:`custom callback <train-custom-callbacks>` for your use case. The callback API
-is shared with Ray Tune.
+.. code-block:: python
 
-Reporting intermediate results
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    print(results.metrics_dataframe)
+    #    epoch  _timestamp  _time_this_iter_s  _training_iteration  time_this_iter_s  ...        node_ip  time_since_restore  timesteps_since_restore  iterations_since_restore warmup_time
```

Comment:
Formatting for this is weird... can we further reduce the columns so it fits on a regularly sized page?